docs/boshdirector.adoc

```diff
@@ -28,114 +40,307 @@ Click on the zone; you'll notice an NS-type entry for the subdomain, as shown be
 .Example Network Configuration: Elastic Runtime
 image::ert_network_config.png[Example Network Configuration,width="50%"]
 
 
 === Assign AZs and Newtorks
 
-. Select the `{env_name}-ops-manager-subnet` in http://docs.pivotal.io/pivotalcf/1-9/customizing/gcp-om-config.html#assign-azs[step 6^]:  that's the subnet in which the bosh director should be installed
+. Browse to the `Assign AZs and Networks` section (left margin).
+
++
+[.thumb]
+.Assign AZs and Networks
+image::assign-azs-networks-gcp.png[Assign AZs and Networks]
 
-Press _Save_.
+. Enter the following values for the specified fields, and accept defaults for all other fields.  Then click `Save`.
 
-=== Apply Changes
++
+.Assign AZs and Networks
+|===
+|Field|Value
+
+|Singleton Availability Zone
+|Select the first value in the dropdown
+
+|Network
+|gcp-management-network
 
-Go back to the installation dashboard and click _Apply Changes_.  It may take up to ten minutes for this to finish. This may be a good time to stretch and grab a cup of coffee.
+|===
 
-In the Google Cloud Console, visit the _Compute Engine -> VM Instances_ section and observe that a new VM will have been created for the BOSH Director.
+=== Apply Changes
 
-Congratulations, you have now deployed the BOSH Director.
+. Be sure all tabs on the left have a green check mark and that the tile is green on the dashboard before continuing to the next exercise.
 
-== Post-Installation
+. **Ask your instructor to validate your settings.**
 
-. in the Ops Manager, select the GCP _tile_.  We need to retrieve some information:
-.. the Ops Manager IP address: navigate to the _Status_ tab, note the IP address for the Ops Manager Director
-.. navigate to the _Credentials_ tab, click the link to the director credentials, note the identity and password displayed in the json response.
+. Browse to the `Installation Dashboard`.  Then click `Apply Changes` to create an instance of Ops Manager Director.
 
-. verify that you can ssh into the Ops Manager with:
 +
-[source.terminal]
-----
-gcloud compute ssh ubuntu@{{env_name}}-ops-manager
-----
+[.thumbs]
+.Apply Changes
+image::director-apply-changes-gcp.png[Apply Changes]
+
 +
-NOTE: The above command dynamically creates a key pair for logging in to the VM
+The installation should take about 10 minutes. You can view the log details as the installation is happening (`Show verbose output`).
+
+. When the installation is successful, you should have receive a message saying "Changes Applied...".
+
 +
-The bosh client is pre-installed on Ops Manager VM.  For security, the bosh director VM is usually not made accessible from the outside world.  The Ops Manager VM serves as a bastion VM from which we can communicate with the bosh director.
+[.thumbs]
+.Changes Applied
+image::director-success-gcp.png[Changes Applied]
 
-. Target, and login to the bosh director:
 +
-[source.terminal]
-----
-bosh --ca-cert /var/tempest/workspaces/default/root_ca_certificate target {{director_ip_address}}
-----
```

Comment:
why was this section redacted?  i believe that interacting with what the ops manager just created for us (bosh directory) helps reinforce what the student actually did.